docs/dev/Lua API.rst

```diff
@@ -3928,6 +3928,211 @@ Examples:
     end
     unit.body.blood_count = math.min(unit.body.blood_max, unit.body.blood_count + healAmount)
 
+DateTime
+========
+
+A module to display, compare, and manipulate dates and times for the dwarven calendar.
+The *DateTime* and *Duration* objects in this module support *addition*, *subtraction*,
+and *<, <=, >, >=, ==* equality operators
+
+DateTime class
+==============
+
+* ``DateTime{ year = ..., year_tick = ..., ticks_per_day = ...}``
+
+  A DateTime object is constructed with the following properties:
+
+  :year: A year. Defaults to zero.
+  :year_tick: A year tick. Defaults to zero.
+  :ticks_per_day: The rate at which time passes in a day.
+                  Defaults to DWARF/fortress mode.
+                  note: if ticks_per_day is given a value <= 1,
+                  it will default to fortress mode.
+  Examples::
+
+    -- create a new instance with year and year_tick set to
+    -- the current game time.
+    local dateTime = DateTime.now()
+    -- construct and set year to 1, year_tick to 1
+    dateTime = DateTime{ year = 1, year_tick = 1 }
+    -- set ticks_per_day to adventure mode
```

Comment:
this comment should also indicate that the time itself will be set to the beginning of year 0